README.md

```diff
@@ -113,6 +113,10 @@ a best practice.
 This check warns if a field isn't explicitly declared as "required" or
 "optional".
 
+### `field.doc.missing`
```

Comment:
This is very minor, but I've tried to keep these alphabetized to make them easier for people to find when scanning the README. Could you move this above `field.id.missing`?

Reply:
It's ok!